ssh/ssh.go

```diff
@@ -155,10 +155,12 @@ func GetExeAndArgs(osEnv config.Environment, gitEnv config.Environment, meta *SS
 		args = append(args, "-batch")
 	}
 
+	isMultiplexing = false
```

Comment:
Since `GetLFSExeAndArgs()` uses the argument name `multiplexing`, I might be inclined to do the same here as well rather than `isMultiplexing`, but I recognize that's a bit of a bikeshed concern.

Reply:
Okay, sounds good.